### ISSUE TITLE
Feature request: Support for Gotify notifications

### ISSUE THREAD

**Issue by qdm12** [qdm12/private-internet-access-docker#131]
1. What's the feature?

Support with a Gotify notification server in order to send notifications to user devices

2. Why do you need this feature?

To know when the container starts/crashes, what port is forwarded etc. More ideas are welcome in the comments!

**Comment by burkasaurusrex** [comment]
It might be cool to integrate with a notification abstraction framework? [caronc/apprise](https://github.com/caronc/apprise) is pretty popular but written in python. [containrrr/shoutrrr](https://github.com/containrrr/shoutrrr) looks like an early golang port though.

**Comment by qdm12** [comment]
Thanks 👍  Good idea, I'll dig into it before adding Gotify support. Anyway, I might prioritize other things (i.e. daily updates of block lists, port forwarding, etc.) before notifications.

**Comment by bozzfozz** [comment]
Here is an example of how to cover several messengers at once

https://github.com/crazy-max/diun/

amqp
discord
gotify
mail
matrix
mqtt
pushover
rocketchat
script
slack
teams
telegram
webhook

**Comment by qdm12** [comment]
Yep https://github.com/crazy-max/diun/ it will be, thanks @bozzfozz !

I will experiment with it in another repo https://github.com/qdm12/ddns-updater/issues/205 and bring it over there afterwards 😉

**Comment by qdm12** [comment]
Actually diun is a bad choice, we'll use https://containrrr.dev/shoutrrr/ instead

**Comment by jerptrs** [comment]
Hey @qdm12,

do you have any updates on this or is this feature not being implemented?

**Comment by nebb00** [comment]
apprise?
